src/com/facebook/buck/util/Escaper.java

```diff
@@ -410,7 +410,7 @@ public static int decodeNumericEscape(
       boolean valid = false;
       for (int i = 0; i < maxCodeLength && pos < escaped.length(); i++) {
         final char digit = (char) table.indexOf(Character.toLowerCase(escaped.charAt(pos)));
-        if (digit == -1 || digit >= base) {
+        if (digit >= base) {
```

Comment:
This looks like it will work correctly, but I would prefer to keep `digit` as an `int` instead.